source/agora/common/Block.d

```diff
@@ -95,3 +95,78 @@ public struct Block
     ///
     public Transaction tx;
 }
+
+/*******************************************************************************
+
+    Creates the genesis block.
+    Note: The output specifies a randomly-generated address,
+    later to be replaced with proper outputs.
+
+    Returns:
+        the genesis block
+
+*******************************************************************************/
+
+public Block getGenesisBlock ( )
+{
+    import agora.common.crypto.Key;
+
+    // note: not using random() as the genesis block should always be the same
+    auto address = KeyPair.fromSeed(
+        Seed.fromString("SCT4KKJNYLTQO4TVDPVJQZEONTVVW66YLRWAINWI3FZDY7U4JS4JJEI4")).address;
```

Comment:
I should make this CTFE-able so we can just use `static immutable`

Reply:
yessss please.